models/edr/alerts/error_models.sql

```diff
@@ -0,0 +1,23 @@
+{{
```

Comment:
cosmetics - al the alert views start with alerts so maybe for the sake of consistency let's rename to alerts_dbt_models or alerts_error_models?

Reply:
Yeah, I renamed to `alerts_dbt_models` but it kind of bothers me that the difference between `alerts_dbt_models` and `alerts_models` really isn't clear from their naming.